eng/service.proj

```diff
@@ -2,12 +2,18 @@
   <PropertyGroup>
     <ServiceDirectory Condition="'$(ServiceDirectory)' == '' and '$(Scope)' != ''">$(Scope)</ServiceDirectory>
     <ServiceDirectory Condition="'$(ServiceDirectory)' == ''">*</ServiceDirectory>
+    <IncludeTests Condition="'$(IncludeTests)' == ''">true</IncludeTests>
+    <IncludeSrc Condition="'$(IncludeSrc)' == ''">true</IncludeSrc>
   </PropertyGroup>
 
-  <ItemGroup>
-    <ExcludeMgmtLib Include="..\sdk\$(ServiceDirectory)\*.Management.*\**\*.csproj;..\sdk\*mgmt*\**\*.csproj" />
-    <ProjectReference Include="..\sdk\$(ServiceDirectory)\**\*.csproj" Exclude="@(ExcludeMgmtLib)"/>
+  <ItemGroup Condition="'$(PackageDirectory)' == ''">
```

Comment:
Please remove this condition we shouldn't have any references to PackageDirectory